code/classes/DMSDocumentCart.php

```diff
@@ -189,4 +217,17 @@ public function getBackend()
     {
         return $this->backend;
     }
+
+    /**
+     * Checks if an item exists within a cart. Returns true (if exists) or false.
+     *
+     * @param $itemID
+     *
+     * @return bool
```

Comment:
Param is missing type, presume `$this->getItem` returns the `DMSDocumentCart` you've hinted below in the inline doc block (which references a non-existent variable)